aten/src/ATen/cpu/vec256/vec256_float_neon.h

```diff
@@ -349,6 +349,16 @@ template <> class Vec256<float> {
     }
     return loadu(tmp);
   }
+  Vec256<float> hypot(const Vec256<float> &b) const {
```

Comment:
Since we don't have a great way to test this, maybe let's exclude the neon vectorized version (for now at least)?